### PR TITLE
Remove create email api function from backend package

### PR DESCRIPTION
**Pull request by Nikpolik**
## Description

Remove createEmail function from `@clerk/backend`.

The equivalent `/emails` Backend API endpoint will also be dropped in the future, since this feature will no longer be available for new instances.

For a brief period it will still be accessible for instances that have used it in the past 7
days.

New instances will get a 403 forbidden response if they try to access it.

## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [x] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

File: .changeset/curvy-suns-roll.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+'@clerk/backend': patch
+---
+
+Remove createEmail function from @clerk/backend.
+
+The corresponding endpoint is also being removed from the API.
```
Comment (nikosdouvlis) [inline .changeset/curvy-suns-roll.md:7]: Discussed with @Nikpolik offline and we will not be providing an alternative method for the time being. 

Can we please add a note informing any customers currently using this to reach out to our support team if they wish to update to v5?

```suggestion
The `emails` endpoint helper and the corresponding `createEmail` method have been removed from the `@clerk/backend` SDK and `apiClint.emails.createEmail` will no longer be available. We will not be providing an alternative method for creating and sending emails directly from our JavaScript SDKs with this release. If you are currently using `createEmail` and you wish to update to the latest SDK version, please reach out to our support team (https://clerk.com/support) so we can assist you.
```

**Review by nikosdouvlis — APPROVED** [review]

File: packages/backend/src/api/__tests__/factory.test.ts [hunk 2]
```diff
@@ -88,39 +86,6 @@ export default (QUnit: QUnit) => {
       );
     });
 
-    test('executes a successful backend API request for a resource that contains data (key related to pagination)', async assert => {
```
Comment (dimkl) [inline packages/backend/src/api/__tests__/factory.test.ts:91]: 🔧  Instead of removing this test, replace the `apiClient.emails.createEmail(body);` with another API call. This test is not related to the `EmailAPI`, it was just used as an example.

**Review by dimkl — CHANGES_REQUESTED** [review]
Please review the comment in the test file

File: packages/backend/src/api/__tests__/factory.test.ts (hunk 2 above)
Comment (Nikpolik) [reply packages/backend/src/api/__tests__/factory.test.ts:91]: Is it possible this test is no longer needed? 🤔 After looking around only `SMSMessage` and `Emails` contain the property `readonly data?: Record<string, any> | null` that is being checked here.

To be completely honest I am not 100% sure what is being tested here  😓  And whats the differences between this test and the first one that `executes a successful backend API request for a single resource and parses the response`

File: .changeset/curvy-suns-roll.md (hunk 1 above)
Comment (Nikpolik) [reply .changeset/curvy-suns-roll.md:7]: Much better message thanks!

File: packages/backend/src/api/__tests__/factory.test.ts (hunk 2 above)
Comment (dimkl) [reply packages/backend/src/api/__tests__/factory.test.ts:91]: You are correct. This case is no longer applicable. let's drop it.

**Review by dimkl — APPROVED** [review]
